client/post-editor/style.scss

```diff
@@ -61,6 +61,32 @@
 	}
 }
 
+.editor__site {
+	display:  flex;
+	background: lighten( $gray, 30% );
+
+	@include breakpoint( ">660px" ) {
+		display: none;
+	}
+
+	.site__title,
+	.site__domain {
+		&::after {
+			@include long-content-fade( $color: lighten( $gray, 30% ) );
+		}
+	}
+
+	.site__home {
```

Comment:
Why is the home icon there anyway? It's there currently in production (only for the mobile editor, not on desktop), so I preserved the same behaviour. If it shouldn't be there I'll remove it.